examples/GTEx/extract_tile_pw_gtex.py

```diff
@@ -237,26 +249,35 @@ def main():
     )
     args = parser.parse_args()
 
-    metadata_csv = args.metadata_csv
-    wsi_dataset_dir = args.wsi_dataset_dir
-    tile_dataset_dir = args.tile_dataset_dir
+    metadata_csv = Path(args.metadata_csv)
+    wsi_dataset_dir = Path(args.wsi_dataset_dir)
+    tile_dataset_dir = Path(args.tile_dataset_dir)
     tile_size = args.tile_size
     n_tiles = args.n_tiles
     level = args.level
     seed = args.seed
     check_tissue = args.check_tissue
 
-    gtex_df = pd.read_csv(metadata_csv)
-    os.makedirs(wsi_dataset_dir)
-
-    sample_ids = gtex_df["Tissue Sample ID"].tolist()
+    try:
+        gtex_df = pd.read_csv(metadata_csv)
+    except FileNotFoundError:
+        print(f"Metadata CSV filepath {metadata_csv} does not exist. Please check.")
```

Comment:
Why just printing the error instead of throwing an exception? I don't feel it's very pythonic, especially if then you return anyway

Reply:
ok, but I don't see much the difference from:
```
print(str(e))
return
```
This is just to retaining the important bit, clarifying imho what happened with no need to show the name of the actual `FileNotFoundError` class, and the rest. 
Program returns, for a seamless exit - no traceback for uncaught exception.

Reply:
ok, my suggestion was simply: 

```python
except FileNotFoundError:
    raise FileNotFoundError(f"Metadata CSV filepath {metadata_csv} does not exist. Please check.")
```

but since it's just a simple script, and probably there won't be anything catching that exception, it is fine anyway

Reply:
I see.. as you prefer, actually. I was just not for showing the traceback at all.
Although I reckon that your suggestion would make pointless catching the exception at all, if you think about it 🤷 
your choice ;)

Reply:
Yep, I closed the conversation because it is fine the way it is :)